README.md

```diff
@@ -22,7 +22,7 @@ git remote add heroku git@heroku.com:my-database-backups.git
 git push heroku master
 ```
 
-Now we need to set some environment variables in order to get the heroku cli working properly using the [heroku-buildpack-toolbet](https://github.com/gregburek/heroku-buildpack-toolbelt.git).
+Now we need to set some environment variables in order to get the heroku cli working properly using the [heroku-buildpack-cli](https://github.com/gregburek/heroku-buildpack-cli.git).
```

Comment:
Thanks but i think the link is wrong: https://github.com/heroku/heroku-buildpack-cli

Reply:
noooooooooooooooooooooooooooooooooooooooooooooooooooooooooo

Reply:
NOooOOooOoOoOOooOOOooOOOOO

fixed.